include/deal.II/particles/particle_handler.h

```diff
@@ -1018,6 +1018,11 @@ namespace Particles
      */
     unsigned int handle;
 
+    /**
+     * Tolerance to be used for GeometryInfo<dim>::is_inside_cell().
+     */
+    const double tol = 1e-12;
```

Comment:
Can we use a more verbose variable name, e.g. `tolerance_inside_cell` or something related? I think that would make it clearer for the reader of the relevant places in the `cc` file (without hovering over the variable to see the documentation).